app/helpers/smart_proxies_helper.rb

```diff
@@ -1,5 +1,5 @@
 module SmartProxiesHelper
-  TABBED_FEATURES = ["Puppet","Puppet CA"]
+  TABBED_FEATURES = ["Puppet", "Puppet CA", "Logs"]
```

Comment:
+1 for working on this separately. The tabbed features list has room for improvements.